packages/@okta/i18n/src/properties/login.properties

```diff
@@ -1048,9 +1048,9 @@ oie.consent.scopes.enduser.title = would like to:
 oie.consent.enduser.accept.label = Yes, it's me
 oie.consent.enduser.deny.label = No, it's not me
 oie.consent.enduser.title = Did you just try to sign in?
-oie.consent.enduser.email.allow.description = To continue, return to the device or window where you requested the email link. You may close this window at any time.
-oie.consent.enduser.deny.description = You may close this window at any time.
-oie.return.to.original.tab = Please return to the original tab.
+oie.consent.enduser.email.verified.title = Email verified!
+oie.return.to.original.tab = Please <span style="font-weight:bold">return</span> to your original screen to continue.
+oie.close.this.window = Close this window anytime.
```

Comment:
Please note that changing key names here will make existing translations for these keys **obsolete**. As such, all languages outside of `en` will fall-back to English instead of being translated.

Given this has already rolled out to new orgs, this is a **breaking change**.

Reply:
I see. I will keep the key to be the same then and update the value instead. Would the translation get updated eventually? Or is it better to introduce new key while keep the current key as well?

Reply:
Yep - translations are usually delivered for our **monthly** releases.

If you reuse the existing key, worst case is customers will see **outdated** translations. In most cases, this is generally acceptable versus falling-back to English. For this reason, we usually advise on key reuse instead of renames.

Reply:
Looks good from the i18n resourcing side! 🚀